client/my-sites/plans-features-main/index.jsx

```diff
@@ -621,6 +623,7 @@ export class PlansFeaturesMain extends Component {
 			hidePlanTypeSelector,
 			is2023PricingGridVisible,
 			planTypeSelectorProps,
+			datastorePlansLoaded = true,
```

Comment:
My understanding is that it intentionally defaults to `true` here since the plans grid is not relying on the data store yet. Is that correct? Either way, it would be better if we leave an inline comment so our folks won't be confused when reading this :)

Reply:
> Is that correct?

This is only here to get the tests passing. It will never default to `true` in a running environment since the HOC feeds the property through. I couldn't get the `withSelect` HOC mocked, at least not without digging further, hence this. I've seen some fix at one place on WooCommerce I think, but a little beyond me right now.

Reply:
> the plans grid is not relying on the data store yet

Well, actually, as of this PR, the grid relies on the data store. It will not render anything until they are fetched in the resolver (hence `datastorePlansLoaded` prop)